tests/image-override-file.bats

```diff
@@ -29,8 +29,8 @@ EOF
   )
 
   run build_image_override_file_with_version "2.1" \
-    "myservice1" "newimage1:1.0.0" "" 0 \
-    "myservice2" "newimage2:1.0.0" "" 0
+    "myservice1" "newimage1:1.0.0" "" 0 0 \
+    "myservice2" "newimage2:1.0.0" "" 0 0
```

Comment:
if you implement my changes, these changes are no longer necessary

Reply:
I tried to get rid of this, but it was a little hard as we still have to shift if its "0". I think we would have to check if it exist and also if its numeric. The argument position should be static in the code so I _think_ it should be safe to only test this flow when using multiple services.

Reply:
in my suggestions I specifically used `${1:-0}` for the case when it was not set but I'm probably missing some corner case or logic... doesn't matter much :)